### PR TITLE
Enable USDT transfers between Statemine and Calamari

### PR DESCRIPTION
**Pull request by Kevingislason**
- Merge upstream polkawallet repo
- Enable USDT transfers between Statemine and Calamari

### CODE REVIEW

**Comment by Kevingislason** [comment]
Note that you may want to review these two commits separately; first commit is from the parent repository, second commit I implemented to enable USDT transfers

**Review by ghzlatarev — APPROVED** [review]

File: src/adapters/manta.ts [hunk 1]
```diff
@@ -77,6 +77,14 @@ export const calamariRoutersConfig: Omit<RouteConfigs, "from">[] = [
       weightLimit: DEST_WEIGHT,
     },
   },
+  {
+    to: "statemine",
```
Comment (zqhxuyuan) [inline src/adapters/manta.ts:81]: ~~should this be Statemint, as Statemine is for Kusama?
or this is just naming, not really matter?~~

Comment (zqhxuyuan) [reply src/adapters/manta.ts:81]: Ok, my bad, I was thought USDT on Polkadot, while this is for USDT on Kusama.

File: src/adapters/manta.ts [hunk 2]
```diff
@@ -86,6 +94,7 @@ export const calamariTokensConfig: Record<string, BasicToken> = {
   LKSM: { name: "LKSM", symbol: "LKSM", decimals: 12, ed: "500000000" },
   KSM: { name: "KSM", symbol: "KSM", decimals: 12, ed: "100000000" },
   MOVR: { name: "MOVR", symbol: "MOVR", decimals: 18, ed: "100000000000000000" },
+  USDT: { name: "USDT", symbol: "USDT", decimals: 6, ed: "10000" }
```
Comment (zqhxuyuan) [inline src/adapters/manta.ts:97]: There're two USDT on Statemine, are you sure we're using 1984, not 19840?

Comment (ghzlatarev) [reply src/adapters/manta.ts:97]: We are using 1984, you can check with chain state.

**Review by zqhxuyuan — APPROVED** [review]

**Pull request merged**